### ISSUE TITLE
Requirement - data entry

### ISSUE THREAD

**Issue by drewdrew122** [racerecords/RaceRecords#8]
The system shall allow the user to enter all required fields without having to use a mouse to change the data entry focus.  Fields should have a logical tab order.

**Comment by mcmillanator** [comment]
Duplicate of #9
